addons/binding/org.openhab.binding.network/src/main/java/org/openhab/binding/network/service/NetworkUtils.java

```diff
@@ -170,6 +170,10 @@ public static boolean nativePing(String hostname, int port, int timeout)
     public static boolean arping(String hostname, int timeout, String networkInterface)
             throws InvalidConfigurationException, IOException, InterruptedException {
         Process proc;
+        if (networkInterface != null) {
+            throw new InvalidConfigurationException(
+                    "Arping requires the network interface to be set (e.g. eth0, wlan0).");
+        }
```

Comment:
Imho this should be done while loading the configuration, not every single time the ping is executed.